rotkehlchen/exchanges/kraken.py

```diff
@@ -1050,3 +1145,137 @@ def history_event_from_kraken(
             returned_events.append(event)
 
         return returned_events, skipped, found_unknown_event
+
+    #################
+    # Futures methods
+    #################
+
+    def _query_futures_api_method(self, method: str, req: dict | None = None) -> dict | str:
```

Comment:
the set of methods that you support is limited right? Type it with `Literal['method_name', 'other_method']` and the linter will check it for you